app/models/driver.rb

```diff
@@ -0,0 +1,8 @@
+class Driver < ApplicationRecord
+  validates :name, presence: true
+  validates :license_number, presence: true
+  validates :mobile_number, presence: true, phone: true
+  validates :email_address, presence: true, email: true
+  validates :age, presence: true
+  validates :status, presence: true
```

Comment:
For status suggest to add validation on inclustion
https://guides.rubyonrails.org/active_record_validations.html#inclusion